src/node_internals.h

```diff
@@ -0,0 +1,90 @@
+// Copyright Joyent, Inc. and other Node contributors.

```

Comment:
Thanks @bnoordhuis, I was looking for some guidance here.  I'll change the copyright headers.